azure-pipelines.yml

```diff
@@ -170,6 +170,51 @@ stages:
       inputs:
         pathtoPublish: '$(Build.ArtifactStagingDirectory)'
         artifactName: wheels
+  - job: manylinux_aarch64_wheels
+    pool:
+      vmImage: "ubuntu-latest"
+    strategy:
+      matrix:
+        Python36:
+          cpython.version: cp36-cp36m
+          manylinux: 'manylinux_aarch64'
```

Comment:
Are these `manylinux` attributes necessary in the matrix? I can't see any reference in the build later on.

Reply:
yes, it is not required. Changes are done.